ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsApi.kt

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api
+
+import com.braintreepayments.api.findeligiblepayments.FindEligiblePaymentsApiResult
+import com.braintreepayments.api.findeligiblepayments.FindEligiblePaymentsApi
+import com.braintreepayments.api.findeligiblepayments.FindEligiblePaymentsApiRequest
+
+/**
+ * [ShopperInsightsApi] is a wrapper class for all api related classes that
+ * [ShopperInsightsClient] depends on.
+ */
+internal class ShopperInsightsApi(
+    private val paymentsApi: FindEligiblePaymentsApi
+) {
+
+    fun findEligiblePayments(request: FindEligiblePaymentsApiRequest): FindEligiblePaymentsApiResult {
```

Comment:
I'm not sure if we're utilizing coroutines for async work. Will we need to make this a suspend function or add a callback for this API call?

Reply:
Yeah we still have not decided yet. We will change this later once we work on API implementation.